plugins/modules/event.py

```diff
@@ -0,0 +1,207 @@
+                    'supported_by': 'XLAB Steampunk'}
+
+DOCUMENTATION = '''
+module: event
+author:
+  - Paul Arthur (@flowerysong)
+  - Aljaz Kosir (@aljazkosir)
+  - Miha Plesko (@miha-plesko)
+  - Tadej Borovsak (@tadeboro)
+short_description: Manages Sensu hooks
+description:
+  - For more information, refer to the Sensu documentation at
+    U(https://docs.sensu.io/sensu-go/latest/reference/events/)
+extends_documentation_fragment:
+  - sensu.sensu_go.base
+options:
+  timestamp:
+    description:
+      - Time that the event occurred in seconds since the Unix epoch.
+    type: int
+  state:
+    description:
+      - Target state of the Sensu object.
+    type: str
+    choices: [ 'present', 'absent' ]
+    default: present
+  entity:
+    description:
+      - Entity object returned from M(entity) or M(entity_info).
+    type: dict
+  check:
+    description:
+      - Check object returned from M(check) or M(check_info).
+    type: dict
+  check_merge:
+    description:
+      - Information about the status and history of the event.
+        U(https://docs.sensu.io/sensu-go/5.13/reference/events/#check-attributes)
+    type: dict
+    suboptions:
+      duration:
+        description: Command execution time in seconds.
+        type: float
+      executed:
+        description: Time that the check request was executed.
+        type: int
+      history:
+        description: Check status history for the last 21 check executions.
+        type: list
+      issued:
+        description: Time that the check request was issued in seconds since the Unix epoch.
+        type: int
+      last_ok:
+        description: The last time that the check returned an OK status (0) in seconds since the Unix epoch.
+        type: int
+      output:
+        description: The output from the execution of the check command.
+        type: str
+      state:
+        description: The state of the check.
+        choices: [ "passing", "failing", "flapping" ]
+        type: str
+      status:
+        description: Exit status code produced by the check.
+        choices: [ "ok", "warning", "critical", "unknown" ]
+        type: str
+      total_state_change:
+        description: The total state change percentage for the check's history.
+        type: int
+'''
+
+EXAMPLES = '''
+- name: Create a simple entity
+  entity:
+    auth:
+      url: http://localhost:8080
+    name: awesome_entity
+    entity_class: proxy
+  register: entity
+
+- name: Create a simple check
+  check:
+    auth:
+      url: http://localhost:8080
+    name: awesome_check
+    command: echo "Check command"
+    output_metric_handlers:
+      - influxdb
+    subscriptions:
+      - checks
+      - also_checks
+    interval: 30
+  register: check
+
+- name: Create event with minimal parameters
+  event:
+    auth:
+      url: http://localhost:8080
+    entity: "{{ entity.object }}"
+    check: "{{ check.object }}"
+    check_merge:
+      output: Check command
```

Comment:
```suggestion
      output: Hello World
```

or something similar. If we put "Check command" it feels like bash command is supposed to be passed here, not the output of it